Makefile

```diff
@@ -37,4 +40,24 @@ run-dimensionality-reduction:
 	docker run --name dimensionality-reduction -p 5002:8000 ifcacomputing/dimensionality-reduction-api
 
 run-model-inference:
-	docker run --name model-inference -p 5003:8000 ifcacomputing/model-inference-api
\ No newline at end of file
+	docker run --name model-inference -p 5003:8000 ifcacomputing/model-inference-api
+    
+mls:
+	docker build -t ghcr.io/grycap/mls-arm-api model_inference_api
+	docker push ghcr.io/grycap/mls-arm-api
+	docker buildx build --platform linux/amd64,linux/arm64 -t ghcr.io/grycap/mls-arm-api --push model_inference_api
+	    
+dds:
+	docker build -t ghcr.io/grycap/dds-arm-api detector_api
+	docker push ghcr.io/grycap/dds-arm-api
+	docker buildx build --platform linux/amd64,linux/arm64 -t ghcr.io/grycap/dds-arm-api --push detector_api
+	    
+emc:
```

Comment:
@vicente87 I think that this should be the unique "new" service. The addition of the others (mls, dds and drs) is completely redundant. Because they are already hosted in ifcacomputing/XXXX-api.

Reply:
The problem is that in order to make the asynchronous invocations we create main-service.py (it's from what was put in main.py) and there we introduce small changes to be able to orchestrate the services well among others. For example we create output files in the drs services in json format so that you can use it well. Also, the input data that comes from the MinIO buckets has to accommodate the functions that you have created and that is why we decided to create the new images. I don't know if I will explain well.